packages/pv-create-component/config.js

```diff
@@ -0,0 +1,12 @@
+// expects pv projects to have the same structure
```

Comment:
can this be overridden?

Reply:
currently not because this should reflect the frontend folder structure in a p!v AEM project and modifying some of them might break the rest (e.g. import paths etc) and imo this can be coupled to the archetype / scaffolding structure and not need / allow further customization.